### PR TITLE
CON-1533 - Confirm training provider Page Changes

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.ProviderRelationships.Web/Controllers/AccountProvidersController.cs [hunk 1]
```diff
@@ -98,6 +98,13 @@ public async Task<ActionResult> Add(AddAccountProviderRouteValues routeValues)
         [Route("add")]
         public async Task<ActionResult> Add(AddAccountProviderViewModel model)
         {
+            if (string.IsNullOrEmpty(model.Choice))
```
Comment (Stephek) [inline src/SFA.DAS.ProviderRelationships.Web/Controllers/AccountProvidersController.cs]: Wouldn't it be better to use ModelState.IsValid and put the [Required] attribute on the model?

Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.ProviderRelationships.Web/Controllers/AccountProvidersController.cs]: as per the suggestion, changes been made using IValidatableObject

File: src/SFA.DAS.ProviderRelationships.Web/Views/Shared/_ValidationSummary.cshtml [hunk 2]
```diff
@@ -3,8 +3,8 @@
 @if (!Model.IsValid)
 {
     <div aria-labelledby="error-summary-title" class="govuk-error-summary" data-module="error-summary" role="alert" tabindex="-1">
-        <h2 class="govuk-error-summary__title  govuk-!-margin-bottom-0" id="error-summary-title">
-            There are errors on this page that need your attention
+        <h2 class="govuk-error-summary__title"  id="error-summary-title">
+            There is a problem
```
Comment (Stephek) [inline src/SFA.DAS.ProviderRelationships.Web/Views/Shared/_ValidationSummary.cshtml]: Should you be changing this global component?

Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.ProviderRelationships.Web/Views/Shared/_ValidationSummary.cshtml]: yes, text is based on the story and  removed the style  based on: https://design-system.service.gov.uk/components/error-summary/

**Pull request merged**